src/SimpleHosts.ts

```diff
@@ -141,7 +141,7 @@ export class SimpleHosts {
                     changed = true;
                 }
             }
-            if (tokens.length < 2) {
+            if (changed && tokens.length < 2) {
```

Comment:
can a single-token lines be valid in a hosts file?

Reply:
Yes as a comment or blank line